account/views.py

```diff
@@ -139,3 +141,82 @@ def edit_profile_view(request):
             {"form": form, "profile": user_profile},
         )
     return HttpResponseNotAllowed(["GET", "POST"])
+
+
+@login_required
+@require_http_methods(["GET"])
+def account_detail_view(request, account_id):
+    """
+    アカウントの詳細を確認するページ
+    """
+
+    if request.method == "GET":
+        account = get_object_or_404(Account, pk=account_id)
+        user_profile = Profile.objects.get(user=account)
```

Comment:
accountとprofileはone to oneなので，
```suggestion
        account = get_object_or_404(Account.objects.select_related("profile"), pk=account_id)
```
だけにして，`profile`は`account.profile.`といった形で使用すると良いと思います。

[この設定](https://qiita.com/fumihiko-hidaka/items/0f619749580da5ad9ce5)などをしてチェックしてもらうと，実際SELECTの回数が減るのが確認できると思います（LOGGINGの設定はPRには含めなくて大丈夫です）